README.md

```diff
@@ -1,207 +1,225 @@
 # SignalFx Java Agent
 
-This is the SignalFx Java Agent, a [JVM
-agent](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
-to automatically instrument your Java application to capture and report
-distributed traces to SignalFx. Enabling the agent is done by downloading the
-[latest JAR](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
-and adding its path to your JVM startup options:
+The SignalFx Java Agent is a
+[Java Virtual Machine](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
```

Comment:
this url is for the instrument package and not the jvm, so suggest "agent" be or be included as the link.